src/cython/gudhi/simplex_tree.pyx

```diff
@@ -68,7 +27,13 @@ cdef class SimplexTree:
     This class is a filtered, with keys, and non contiguous vertices version
     of the simplex tree.
     """
-    cdef Simplex_tree_interface_full_featured * thisptr
+    # unfortunately 'cdef public Simplex_tree_interface_full_featured* thisptr' is not possible
+    # Use intptr_t instead to cast the pointer
+    cdef public intptr_t thisptr
+
+    # Get the pointer casted as it should be
+    cdef Simplex_tree_interface_full_featured* get_ptr(self):
+        return <Simplex_tree_interface_full_featured*>(self.thisptr)
```

Comment:
I wonder if we are allowed to define a property here. If so, it would hide a bit more the fact that thisptr is not actually a `Simplex_tree_interface_full_featured*`. But `get_ptr` is fine.